utils/mgmt_system.py

```diff
@@ -71,6 +77,32 @@ def get_search_filter_spec(self, *args, **kwargs):
 
         return pfs
 
+    def __getattribute__(self, attr):
+        # fetch the attribute using parent class to avoid recursion
```

Comment:
That is addressed as well now by `# any callable (except 'login') is protected against unexpected logout`